Nef_3/include/CGAL/Nef_3/bounded_side_3.h

```diff
@@ -70,21 +70,27 @@ Bounded_side bounded_side_3(IteratorForward first,
 
 
 
+  auto apv = approx(plane.orthogonal_vector());
+
+  int dir = 0;
+  double max = CGAL::abs(apv.x().sup());
```

Comment:
Is that granted that the kernel used by this code is always a filtering kernel from CGAL (either `Epick` or `Epeck`)?

What will happen if the user chooses `CGAL::Simple_cartesian<CGAL::Gmpq>` as the kernel?

Reply:
It is even wrong for `Epick` as the intervals of the three coordinates might all contain 0.   I'll fix it today.

Reply:
Is it possible to make it a fuctor of the Kernel or is that overkill?

Reply:
It is not an overkill but the only clean way to do it.   I  just added a functor that finds any dimension different from zero  (that's also what was done before),